third-party/realdds/src/dds-device-server.cpp

```diff
@@ -138,6 +138,9 @@ static void on_discovery_stream_header( std::shared_ptr< dds_stream_server > con
         { "n-options", stream->options().size() },
         { "options" , stream_options }
         } );
+    LOG_DEBUG( "-----> JSON = " << stream_options_message.json_data().dump() );
+    LOG_DEBUG( "-----> JSON size = " << stream_options_message.json_data().dump().length() );
+    LOG_DEBUG( "-----> CBOR size = " << json::to_cbor( stream_options_message.json_data() ).size() );
```

Comment:
Is this OK with performance? to_cbor? just for printing

Reply:
For the development. We wanted to limit the messages to 4K and we prefer to use JSON over CBOR. This prints help us identify if we are under 4K and how much bandwidth benefit we can get from using CBOR (at the cost of readable debug logs).
Will be removed later...